FindAllGroupsInFolders.groovy

```diff
@@ -0,0 +1,39 @@
+import nectar.plugins.rbac.groups.*  
+import com.cloudbees.hudson.plugins.folder.*
+import com.cloudbees.jenkins.plugins.foldersplus.*
+import com.cloudbees.hudson.plugins.folder.properties.FolderProxyGroupContainer
+
+//Parent folder name to start with
+String folderName = 'EmptyFolder'
+folderItem = Jenkins.instance.getAllItems(Folder.class).find{it.name.equals(folderName)}
```

Comment:
I haven't been able to confirm this locally, but could you do `Jenkins.instance.getAllItems(FolderProxyGroupContainer.class)` instead of searching everywhere manually?